sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestDefineListParameter.java

```diff
@@ -58,7 +58,7 @@ public void setUp() {
 
     @Test
     public void testWithBindList() {
-        TestDao testDao = handle.attach(TestDao.class);
+        final TestDao testDao = handle.attach(TestDao.class);
```

Comment:
why this change?

Reply:
`VariableDeclarationUsageDistance` check, same as above